.github/workflows/release-push-image.yaml

```
@@ -45,6 +50,39 @@ jobs:
       java-distribution: ${{ inputs.java-distribution || 'temurin' }}
       java-version: ${{ inputs.java-version || '21.0.6' }}
 
+  package-block-node-protobuf:
+    timeout-minutes: 5
+    needs: [check-gradle]
+    runs-on: hiero-block-node-linux-medium
+
+    steps:
+      - name: Harden Runner
+        uses: step-security/harden-runner@0634a2670c59f64b4a01f0f96f84700a4088b9f0 # v2.12.0
+        with:
+          egress-policy: audit
+
+      - name: Checkout repository
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+        with:
+          fetch-depth: 0
+
+      - name: Extract version
+        id: extract_version
+        run: |
+          VERSION=$(cat version.txt)
+          echo "VERSION=${VERSION}" >> $GITHUB_ENV
+
+      - name: Produce artifact
+        working-directory: protobuf
+        run: scripts/build-bn-proto.sh -t "efb0134e921b32ed6302da9c93874d65492e876f" -v ${{ env.VERSION }} -o "block-node-protobuf" # CN 0.62.2
+
+      - name: Upload artifact
+        if: ${{ env.INPUT_PUBLISH == 'true' }}
```

I don't see `INPUT_PUBLISH` being set anywhere. I'd use the variable directly from the input, like this:
```suggestion
        if: ${{ github.event_name != 'workflow_dispatch' || github.event.inputs.publish == 'true' }}
```

---

Updated with suggestion